modules/evs/fsw/inc/cfe_evs_msg.h

```diff
@@ -0,0 +1,1287 @@
+**       This command sets the logging mode to the command specified value.
+**
+**  \cfecmdmnemonic \EVS_SETLOGMODE
+**
+**  \par Command Structure
+**       #CFE_EVS_SetLogModeCmd_t
+**
+**  \par Command Verification
+**       Successful execution of this command may be verified with
+**       the following telemetry:
+**       - \b \c \EVS_CMDPC - command execution counter will
+**       increment
+**       - The generation of #CFE_EVS_LOGMODE_EID debug event message
+**
+**  \par Error Conditions
+**      This command may fail for the following reason(s):
+**      - Invalid SB message (command) length
+**      - Invalid MODE selected
+**
+**       Evidence of failure may be found in the following telemetry:
+**       - \b \c \EVS_CMDEC - command error counter will increment
+**       - An Error specific event message
+**
+**  \par Criticality
+**       Setting the event logging mode is not particularly hazardous, as the
+**       result may be saving valuable event data.  However, inappropriately
+**       setting the log mode could result in a loss of critical information.
+**      Note: the event log is a back-up log to the on-board recorder.
+**
+**  \sa #CFE_EVS_WRITE_LOG_DATA_FILE_CC, #CFE_EVS_CLEAR_LOG_CC
+*/
+#define CFE_EVS_SET_LOG_MODE_CC 19
+
+/** \cfeevscmd Clear Event Log
+**
+**  \par Description
+**       This command clears the contents of the local event log.
+**
+**  \cfecmdmnemonic \EVS_CLRLOG
+**
+**  \par Command Structure
+**       #CFE_EVS_ClearLogCmd_t
+**
+**  \par Command Verification
+**       Successful execution of this command may be verified with
+**       the following telemetry:
+**       - \b \c \EVS_CMDPC - command execution counter will
+**       increment
+**
+**  \par Error Conditions
+**      This command may fail for the following reason(s):
+**      - Invalid SB message (command) length
+**
+**       Evidence of failure may be found in the following telemetry:
+**       - \b \c \EVS_CMDEC - command error counter will increment
+**       - An Error specific event message
+**
+**  \par Criticality
+**       Clearing the local event log is not particularly hazardous, as the
+**      result may be making available space to record valuable event data.
+**      However, inappropriately clearing the local event log could result
+**      in a loss of critical information.  Note: the event log is a back-up
+**      log to the on-board recorder.
+**
+**  \sa #CFE_EVS_WRITE_LOG_DATA_FILE_CC, #CFE_EVS_SET_LOG_MODE_CC
+*/
+#define CFE_EVS_CLEAR_LOG_CC 20
+/** \} */
+
+/* Event Type bit masks */
+#define CFE_EVS_DEBUG_BIT       0x0001
+#define CFE_EVS_INFORMATION_BIT 0x0002
+#define CFE_EVS_ERROR_BIT       0x0004
+#define CFE_EVS_CRITICAL_BIT    0x0008
+
+/* Output Port bit masks */
+#define CFE_EVS_PORT1_BIT 0x0001
+#define CFE_EVS_PORT2_BIT 0x0002
+#define CFE_EVS_PORT3_BIT 0x0004
+#define CFE_EVS_PORT4_BIT 0x0008
+
+/* EVS Log Modes */
+#define CFE_EVS_LOG_OVERWRITE 0
+#define CFE_EVS_LOG_DISCARD   1
```

Comment:
this seems repeat of:
modules/core_api/fsw/inc/cfe_evs_extern_typedefs.h 
lines: 69-81 (yes names are slightly different and it looks dependent on coding standard)
which hints at the fact that information is not located where people can find it, so they redefine on own

Reply:
Definitely address

Reply:
https://github.com/nasa/cFE/issues/1446